.npmrc

```diff
@@ -0,0 +1,3 @@
+ignore-workspace-root-check=true
+shamefully-hoist=true
```

Comment:
Can we avoid `shamefully-hoist`?

Reply:
After remove it, vitepress build get [some error](https://github.com/WestonSeyler/unplugin/actions/runs/8197271878/job/22418976460#step:6:109) throw by `twoslash`.

Reply:
Because `unplugin` itself wasn't built.